### DISCUSSION TITLE
Alert output formatting

### DISCUSSION THREAD

**Discussion by Raks2110** [jertel/elastalert2#94] [Q&A] [answered]
type:fluent-bit-logs      tags:_timestampparsefailure, _grokparsefailure stream:stderr 
log:*1|ERRORServer handshake failed, err[remote error: tls: bad certificate] @version:1 
Pod:co @timestamp:Apr 27, 2021 @ 08:30:04.505 
Namespace:default 
Container_ID:con time:Apr 27, 2021 @ 08:30:04.456 
_id:vaFEEXkBMG7H_dQ4_QSA _
type:doc 
_index:cloud-2021.04.27 _score:

This is the standard format of log observed on kibana dashboard , there are many fields as mentioned above , but on my email or slack i want the elastalert to send only few fields data like , "log  ,  pod , container" how can i filter out only those fields and get an alert

**Comment by ferozsalam** [comment, marked as answer]
The documentation [here](https://elastalert2.readthedocs.io/en/latest/ruletypes.html#alert-content) covers it.

**Reply by Raks2110** [reply]
Warning


Warning
output is like this

**Reply by ferozsalam** [reply]
You also need an `alert_text` field.

**Reply by Raks2110** [reply]
got it thank you ,
